### PR TITLE
Add speakers link

### PR DESCRIPTION
**Pull request by Ploypiti**
Issue #85 
[Preview](https://ploypiti.github.io/2017.fossasia.org/)

### CODE REVIEW

**Review by akshatnitd — COMMENTED** [review]
The changes looks good. Please provide a deployment link, so that we can see the changes :)

File: index.html [hunk 1]
```diff
@@ -1036,7 +1041,8 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://www.alexcg.net"><i class="icon fa fa-external-link"></i></a>
-											<a target="default" href="https://twitter.com/alexcg"><i class="icon social_twitter"></i></a>
+											<a target="default" href="https://github.com/daminisatya"><i class="icon fa fa-github"></i></a>
```
Comment (anshumanv) [inline index.html]: Wrong link.

**Review by anshumanv — DISMISSED** [review]
Check comments

File: index.html (hunk 1 above)
Comment (Ploypiti) [reply index.html]: Shoot, sorry.

Comment (Ploypiti) [reply index.html]: Fixed.

File: index.html [hunk 2]
```diff
@@ -1036,7 +1041,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://www.alexcg.net"><i class="icon fa fa-external-link"></i></a>
-											<a target="default" href="https://twitter.com/alexcg"><i class="icon social_twitter"></i></a>
+											<a target="default" href="https://github.com/alexcg1"><i class="icon social_twitter"></i></a>
```
Comment (anshumanv) [inline index.html]: Github link in twitter icon can be misleading, please don't remove the twitter icon, add github icon separately.

File: index.html [hunk 3]
```diff
@@ -1041,6 +1041,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://www.alexcg.net"><i class="icon fa fa-external-link"></i></a>
+											<a target="default" href="https://twitter.com/alexcg"><i class="icon social_twitter"></i></a>
```
Comment (anshumanv) [inline index.html]: Please use class as `fa fa-twitter`

**Review by vaibhavsingh97 — APPROVED** [review]
LGTM

**Comment by vaibhavsingh97** [comment]
@Ploypiti PLease squash your commits and change your commit message to `Fixes #<issue-number>: commit message>`

File: index.html (hunk 3 above)
Comment (anshumanv) [reply index.html]: Okay, working either way.

**Review by anshumanv — APPROVED** [review]
LGTM

File: index.html [hunk 4]
```diff
@@ -671,6 +671,7 @@ <h1>Featured Speakers</h1>
 									<img class="background-image" alt="Philip Paeps" src="img/PhilipPaeps.jpg">
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
+											<a target="default" href="https://github.com/ppaeps"><i class="icon fa fa-github"></i></a>
 											<a target="default" href="https://wiki.freebsd.org/PhilipPaeps"><i class="icon fa fa-external-link"></i></a>
                       <a target="default" href="https://github.com/ppaeps"><i class="icon social_github"></i></a>
```
Comment (NaviRocker) [inline index.html:676]: Please fixe the Indentation.

**Review by saarthakchats — APPROVED** [review]
Good work!

File: index.html (hunk 4 above)
Comment (Ploypiti) [reply index.html:676]: ℱⅈℵℯⅆ

**Review by abishekvashok — APPROVED** [review]

**Comment by Ploypiti** [comment]
@akshatnitd @vaibhavsingh97 Merge, please? °^°

File: index.html (hunk 4 above)
Comment (nzec) [reply index.html:676]: @Ploypiti Don't use that font 😆

**Review by nzec — APPROVED** [review]
Awesome! 🚀

**Review by akshatnitd — APPROVED** [review]

**Pull request merged**